articles/quickstart/native/android/_includes/_login.md

```diff
@@ -1,11 +1,30 @@
 ## Start the Authentication
 
-In the login method create a new `Auth0` instance to hold the credentials. If you've added the `R.string.com_auth0_client_id` and `R.string.com_auth0_domain` String resources, you'll be able to use the constructor that receives an android Context. If you prefer to hardcode them, use the constructor that receives both strings. Then by using the `WebAuthProvider` class authenticate with any connection enabled on your client in the Auth0 dashboard.
+The [Auth0 hosted login page](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using the Auth0 hosted login page for the best experience, best security and the fullest array of features.
 
-To ensure Open ID Connect compliant responses you must either request an `audience` or enable the **OIDC Conformant** switch in your Auth0 dashboard under `Client / Settings / Advanced OAuth`. You can read more about this [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
+::: note
+You can also embed the Lock widget directly in your application. If you use this method, some features, such as single sign-on, will not be accessible. 
+To learn how to embed the Lock widget in your application, follow the [Embedded Login sample](https://github.com/auth0-samples/auth0-android-sample/tree/embedded-login/01-Embedded-Login).
+:::
 
+In the `login` method, create a new instance of the `Auth0` object to hold user credentials. 
 
-After calling `WebAuthProvider#start` the browser will launch and show **Lock**, and the final result will be received in the callback that you pass.
+You can use a constructor that receives an Android Context if you have added the following String resources: 
+* `R.string.com_auth0_client_id`
+* `R.string.com_auth0_domain`
+
+If you prefer to hardcode the resources, use the constructor that receives both strings. Then, use the `WebAuthProvider` class to authenticate with any connection you enabled on your client in the  [Auth0 dashboard](${manage_url}/#/).
+
+You need to make sure you get a reponse compliant with the OpenID Connect protocol. You can choose between two options:
+
+* Set the audience
+* Turn on the **OIDC conformant** switch in your Auth0 dashboard
+  		  
+::: note
+To turn on the **OIDC conformant** switch, in your [Client Settings](${manage_url}/#/applications/${account.clientId}/settings), click on **Show Advanced Settings** > **OAuth**. To learn more, read the [net flows documentation](/api-auth/intro#how-to-use-the-new-flows).
+:::
+
+After you call the `WebAuthProvider#start` function, the browser launches and shows the **Lock** widget. The callback URL contains the final result of the authentication process. 
```

Comment:
"After you call the `WebAuthProvider#start` function, the browser launches and shows the **Lock** widget. Once the user authenticates, the callback URL will be called with the final result of the authentication process. "